assets/monero.conf.template

```diff
@@ -1,41 +1,41 @@
 #
 # monero.conf configuration file.  Things beginning with # are comments.
 #
-# See https://monerodocs.org/interacting/monerod-reference/
+# See https://getmonero.dev/interacting/monerod
 
 # Do not require tty as in foreground mode: "Helpful when running in a container."
 # This keeps monero from crashing because no tty is assigned.
 # We are commenting it here because it *must* be invoked from the command line via `monerod --non-interactive`
 #non-interactive=1
 
 # Data directory (blockchain db and indices)
-data-dir=/data/.bitmonero  # Remember to create the monero user first
+data-dir=BITMONERO_DIR          # Remember to create the monero user first
 
 # Log file
 log-level=0,blockchain:INFO
-log-file=/data/.bitmonero/monero.log
-max-log-file-size=10000000            # Currently 10MB. 0 = Prevent monerod from managing the log files, if we want logrotate to take care of that
+log-file=MONERO_LOG
+max-log-file-size=10000000     # Our default: 10MB. 0 = Prevent monerod from managing the log files, if we want logrotate to take care of that
 max-log-files=2
 
 # P2P full node
-p2p-bind-ip=0.0.0.0            # Bind to all interfaces (the default)
-p2p-bind-port=18080            # Bind to default port
+p2p-bind-ip=MONEROD_BIND_IP            # Bind to all interfaces (the default)
+p2p-bind-port=MONERO_P2P_PORT            # Bind to default port
 
-out-peers=ADV_P2P_MAXNUMOUTPEERS    # This will enable much faster sync and tx awareness; the default 8 is suboptimal nowadays [defacto default: 64]
-in-peers=ADV_P2P_MAXNUMINPEERS      # The default is unlimited; we prefer to put a cap on this ["suggested": 128]
+out-peers=ADV_P2P_MAXNUMOUTPEERS         # Larger values will enable much faster sync and tx awareness
+in-peers=ADV_P2P_MAXNUMINPEERS          # The default is unlimited; we prefer to put a cap on this ["suggested": 128]
 
-limit-rate-up=RATELIMIT_KBPSUP       # Default 2048 kB/s; contribute more to p2p network
-limit-rate-down=RATELIMIT_KBPSDOWN   # Default 8192 kB/s; allow for faster initial sync
+limit-rate-up=RATELIMIT_KBPSUP        # Default 2048 kB/s; contribute more to p2p network
+limit-rate-down=RATELIMIT_KBPSDOWN      # Default 8192 kB/s; allow for faster initial sync
 
 # RPC open node
-rpc-bind-ip=0.0.0.0            # Unrestricted RPC IP and port. 0.0.0.0=Bind to all interfaces [So we can connect a wallet on the LAN]
-rpc-bind-port=18081            # Bind on default port
+rpc-bind-ip=MONEROD_BIND_IP            # Unrestricted RPC IP and port. 0.0.0.0=Bind to all interfaces [So we can connect a wallet on the LAN]
```

Comment:
```suggestion
rpc-bind-ip=MONEROD_LOCAL_HOST            # Unrestricted RPC IP and port. 
```
Leave this alone

Reply:
You leave it alone!  How can any other container or any wallet access the monero RPC if it only binds to localhost?

Reply:
Using the restricted rpc bind
its unsafe to open the unrestricted bind

Reply:
Are you saying there is no reason whatsoever to use the unrestricted RPC interface?  Why does it exist?

Reply:
to stop the node, to control monerod mining, to modify the blockchain file (pop blocks etc) and a few other commands

Reply:
> xmrig is another service on StartOS.

xmrig can use restricted rpc

Reply:
OK.  I was trying to open it up to any conceivable use case.  But since you are so confident in saying it's not needed, I am not opposed to turning it off.  Then anybody who actually wants it can make a feature request in the future.